.github/renovate.json

```diff
@@ -3,7 +3,59 @@
     "github>rancher/renovate-config#release"
   ],
   "baseBranches": [
-    "main"
+    "main", 
+    "release/v1.1", 
+    "release/v1.2", 
+    "release/v1.3"
   ],
-  "prHourlyLimit": 2
+  "prHourlyLimit": 2,
```

Comment:
We may need to increase this given the different base branches. Let's revisit this as we go.

Reply:
sure